### PR TITLE
Added working HT to read words from file/text and count them

### PR DESCRIPTION
**Pull request by selestiel**
unordered map not implemented

### CODE REVIEW

File: HT_RANDOM_TEXT/functions.cpp [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    int count{0};
+    std::string word_f;
+    std::ifstream file;
+    file.open(filename);
+    if(file.is_open()){
+        while(!file.eof()){
+            while(file >> word_f){
+                if(word_f.size() > 0){
+                    std::cout << word_f << std::endl;
+                    count++;
```
Comment (smay1613) [inline HT_RANDOM_TEXT/functions.cpp:15]: Wrong implementation.

File: HT_RANDOM_TEXT/functions.cpp [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+void read_text(std::string m_text)
+{   int count{0};
+    getline(std::cin, m_text);
+    std::istringstream iss(m_text);
+    std::string word_out;
+    std::cout << std::endl;
+    while(iss >> word_out){
+        std::cout << word_out << std::endl;
+        count++;
```
Comment (smay1613) [inline HT_RANDOM_TEXT/functions.cpp:35]: You shoudln't count words manually.

**Review by smay1613 — CHANGES_REQUESTED** [review]

**Comment by selestiel** [comment]
I'll leave HT as it is with no further mods for now....

**Comment by selestiel** [comment]
please review Text_mapped

**Pull request closed without merging**